modules/nf-core/verkko/meta.yml

```diff
@@ -0,0 +1,52 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/yaml-schema.json
+name: "verkko"
+description: hybrid genome assembly pipeline developed for telomere-to-telomere assembly of PacBio HiFi and Oxford Nanopore reads
+keywords:
+  - assembly
+  - telomere-to-telomere
+  - pacbio hifi
+  - nanopore ultra-long
```

Comment:
This module is not supporting nanopore ultra-long at this stage.

Reply:
```suggestion
```

Reply:
Yeah, that's upsetting because ONT works on my local, but the tests will crash :(